src/screens/CreatePost/index.js

```diff
@@ -923,24 +932,27 @@ const styles = StyleSheet.create({
     paddingBottom: 13
   },
   userTarget: {
-    color: colors.bondi_blue,
+    color: COLORS.holyTosca,
     fontSize: 14,
     fontFamily: fonts.poppins[400]
   },
   height: (height) => ({
     height
   }),
   reminderContainer: {
-    backgroundColor: colors.blue,
+    backgroundColor: COLORS.holyTosca,
     alignItems: 'center',
     justifyContent: 'center',
     paddingVertical: 7,
     borderBottomLeftRadius: 10,
     borderBottomRightRadius: 10
   },
   whiteText: {
-    color: 'white',
+    color: COLORS.white,
     fontSize: normalizeFontSize(10),
     textAlign: 'center'
-  }
+  },
+  buttonPost: (color) => ({
+    backgroundColor: color.primary
+  })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good as they replace hardcoded color values with constants from a theme file which improves maintainability.
```diff
-    color: colors.bondi_blue,
+    color: COLORS.holyTosca,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->